assignment/index.html

```diff
@@ -63,9 +65,45 @@
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
+      let healthCentersArray = [['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']]
+      for (var i=1; i<healthCenters.length; i++) {
+          var nestedarry = [parseFloat(`${healthCenters[i].LNG}`),
+                   parseFloat(`${healthCenters[i].LAT}`),
+                      `${healthCenters[i].OBJECTID}`,
+                      `${healthCenters[i].NAME}`,
+                      `${healthCenters[i].ORGANIZATION}`,
+                   Number(`${healthCenters[i].ZIP}`),
```

Comment:
We encourage working together.
But I have seen this exact same strategy a few times and it's definitely not quite the right way of solving it, and I just want to make sure you are all learning.